packages/core/src/tracing/cls.js

```diff
@@ -160,7 +160,11 @@ class InstanaSpan {
   }
 
   transmit() {
+    // eslint-disable-next-line no-console
+    console.log('transmit', this, this.transmitted, this.manualEndMode);
```

Comment:
Okay, I see now. The "review-and-merge" label lead me to believe that you planned to merge this PR as-is but there are still a couple more CI-troubleshooting console.logs in here, also in sdk.js, so looks like I was mistaken about that. I'll stop commenting on those :-)

Reply:
Yes correct :)